deploy/maistra-operator.yaml

```diff
@@ -10653,31 +10653,6 @@ spec:
       labels:
         name: istio-operator
       annotations:
-        olm.relatedImage.v1_0.3scale-istio-adapter: quay.io/3scale/3scale-istio-adapter:v1.0.0
-        olm.relatedImage.v1_0.citadel: docker.io/maistra/citadel-ubi8:1.0.8
-        olm.relatedImage.v1_0.cni: docker.io/maistra/istio-cni-ubi8:1.0.8
-        olm.relatedImage.v1_0.galley: docker.io/maistra/galley-ubi8:1.0.8
-        olm.relatedImage.v1_0.grafana: docker.io/maistra/grafana-ubi8:1.0.8
-        olm.relatedImage.v1_0.mixer: docker.io/maistra/mixer-ubi8:1.0.8
-        olm.relatedImage.v1_0.pilot: docker.io/maistra/pilot-ubi8:1.0.8
-        olm.relatedImage.v1_0.prometheus: docker.io/maistra/prometheus-ubi8:1.0.8
-        olm.relatedImage.v1_0.proxy-init: docker.io/maistra/proxy-init-centos7:1.0.8
-        olm.relatedImage.v1_0.proxyv2: docker.io/maistra/proxyv2-ubi8:1.0.8
-        olm.relatedImage.v1_0.sidecar-injector: docker.io/maistra/sidecar-injector-ubi8:1.0.8
```

Comment:
As far as I remember correctly, we didn't remove these images intentionally, so I guess we shouldn't remove them and the images below as well.

Reply:
@jwendell could you confirm?

Reply:
@jewertow I haven't found a mail or doc about keeping those annotations. Do we still need them ?

Reply:
not sure it was intentional. let's remove both